src/vm/dllimportcallback.cpp

```diff
@@ -136,40 +136,6 @@ void UM2MThunk_Wrapper(LPVOID ptr) // UM2MThunk_Args
     END_CALL_TO_MANAGED();
```

Comment:
You can delete `UM2MThunk_Wrapper` and `UM2MThunk_WrapperHelper` from everywhere as well.

Reply:
Nice